### ISSUE TITLE
Text appears Italics

### ISSUE THREAD

**Issue by sanishkarkee** [sreetamdas/karma#21]
Hands down one of the best theme I have used till date. But one thing is bothering me a bit. There should be options to chose normal, italics texts. .HTML, Js are working fine but all the texts in .css are italics which makes it a bit hard to read. 
![image](https://github.com/user-attachments/assets/1a0cedde-e5a1-472f-afd0-90886bccfdeb)
![Uploading Screenshot 2024-08-31 131251.png…]()

**Comment by sreetamdas** [comment]
@sanishkarkee thanks so much! Do you suggest a version without _any_ italics?

**Comment by sanishkarkee** [comment]
Yes that would be great also if possible can you make the background a
little less black coz its way too black . I mean different versions of
background would be great.
